examples/one-2-one/README.md

```diff
@@ -0,0 +1,3 @@
+# one2one
```

Comment:
The README needs to be improved before this patch is landed. See our [README guidelines](https://loopback.io/doc/en/contrib/README-guidelines.html).

Reply:
I will remove my hasOne example as per your notification below.